src/components/programsBottomHeaderArea.js

```diff
@@ -0,0 +1,94 @@
+import React from "react"
+import { useStaticQuery, graphql } from "gatsby"
+import "./programsBottomHeaderArea.scss"
+import ProgrammingCard from "./programmingCard"
+import ProgrammingCardExpandable from "./programmingCardExpandable"
+
+export default function ProgramsBottomHeaderArea() {
+  const data = useStaticQuery(graphql`
+    {
+      allContentfulProgram {
+        edges {
+          node {
+            title
+            summary {
+              content {
+                content {
+                  value
+                }
+              }
+            }
+            contentful_id
+          }
+        }
+      }
+    }
+  `)
+
+  const cardsData = data.allContentfulProgram.edges.reduce((acc, cur) => {
+    const { title, contentful_id } = cur.node
+    let summary = cur.node.summary.content[0].content.reduce(
+      (summary, curSummary) => {
+        summary.push(curSummary.value)
+        return summary
+      },
+      []
+    )
+
+    summary = summary.join("")
+
+    const data = {
+      title,
+      summary,
+      id: contentful_id,
+    }
+
+    acc.push(data)
+    return acc
+  }, [])
+
+  const mediaCard = cardsData.find(card =>
+    card.title.includes("Media ArtsMatter")
+  )
+  const la2030Card = cardsData.find(card => card.title.includes("LA2030"))
+  const filmIndependentCard = cardsData.find(card =>
+    card.title.includes("Film Independent")
+  )
+  const spongebobCard = cardsData.find(card => card.title.includes("SpongeBob"))
+  const wonderParkCard = cardsData.find(card =>
+    card.title.includes("Wonder Park")
+  )
+  const sherlockGnomesCard = cardsData.find(card =>
+    card.title.includes("Sherlock Gnomes")
+  )
+
+  return (
+    <>
+      <div className="current-programming">
+        <div className="current-programming__title">current programming</div>
+        <div className="current-programming__cards">
+          <ProgrammingCard
```

Comment:
Let's talk about this. Ideally we should be looping through this so that it doesn't require any manual updates when new programs are added.